Samples/kotlin/src/main/java/com/phunware/kotlin/sample/OffRouteActivity.kt

```diff
@@ -64,6 +74,8 @@ class OffRouteActivity : AppCompatActivity(), OnPhunwareMapReadyCallback,
     private val offRouteDistanceThreshold = 10.0 //distance in meters
     private val offRouteTimeThreshold: Long = 5000 //time in milliseconds
     private var offRouteTimer: Timer? = null
+    private var previousTimeDialogDismissed = 0L
+    private val timeBetweenDialogPromptThreshold = 10000 //time in milliseconds
```

Comment:
Nitpick: Maybe make this a constant?

Reply:
Nitpick: Can we add some comments around this to describe which behavior it controls?

Reply:
good call will do